### PR TITLE
34 create forms with ability to add edit delete competitorsparticipants

### PR DESCRIPTION
**Pull request by molloyconor511**
Add competitors styling

### CODE REVIEW

File: src/server/controllers/admin.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ const authenticateAdmin = async (req, res) => {
         return res.status(HTTP_RESPONSE.CREATED.CODE).json({ data: selectedAdmin, token: token });
     } catch (error) {
         console.log(error);
-        return res.status(HTTP_RESPONSE.INTERNAL.CODE).json({ error: HTTP_RESPONSE.INTERNAL.MESSAGE });
+       // return res.status(HTTP_RESPONSE.INTERNAL.CODE).json({ error: HTTP_RESPONSE.INTERNAL.MESSAGE });
```
Comment (polarbear23) [inline src/server/controllers/admin.js:70]: uncomment

File: src/client/components/Competitors.jsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
+import React, { useState } from "react";
+
+import "./styling/competitors.css";
+
+const Competitors = () => {
```
Comment (bravint) [inline src/client/components/Competitors.jsx:5]: change line to 

<code>export const Competitors = () => {</code>

to make it consistent with other components

File: src/client/components/Competitors.jsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        className="submit-completed-competition"
+        onClick={handlePostSubmit}
+      >
+        Create New Competition
+      </button>
+    </div>
+  );
+};
+
+export default Competitors;
```
Comment (bravint) [inline src/client/components/Competitors.jsx:132]: then remove this line

File: src/client/App.jsx [hunk 4]
```diff
@@ -8,9 +8,9 @@ import { Competition } from './components/Competition';
 import { Season } from './components/Season';
 import { CreateSeason } from './components/CreateSeason';
 
+import Competitors from './components/Competitors';
```
Comment (bravint) [inline src/client/App.jsx:11]: 3. This needs changing to

<code>import  { Competitors } from './components/Competitors';</code>

**Review by polarbear23 — APPROVED** [review]

**Review by bravint — APPROVED** [review]
nice work Conor

File: src/client/components/Competitors.jsx [hunk 5]
```diff
@@ -0,0 +1,133 @@
+import React, { useState } from "react";
+
+import "./styling/competitors.css";
+
+export const Competitors = () => {
+  const [newCompetitionName, setNewCompetitionName] = useState("");
+  const [competitors, setCompetitors] = useState([]);
+  const [newCompetitorName, setNewCompetitorName] = useState("");
```
Comment (glowkeeper) [inline src/client/components/Competitors.jsx:8]: I don't mind this so much (so I won't let it stop me approving), but I reckon I'd prefer a single competitor object with a number of properties ;) That way, you'd only need a single `setCompetitor`, which you'd set with something like `setCompetitor({...competitor, [key]: value})`

File: src/client/components/Season.jsx [hunk 6]
```diff
@@ -14,10 +14,10 @@ export const Season = () => {
             
             {/*dropdown needs styled*/}
             <select className="select-round-dropdown" name="rounds">
-                <option value="1">{`Mario Circuit`}</option>
-                <option value="2">{`Coconut Mall`}</option>
-                <option value="3">{`DK Summit`}</option>
-                <option value="4">{`Wario's Gold Mine`}</option>
+                <option className="dropdown-option" value="1">{`Mario Circuit`}</option>
```
Comment (glowkeeper) [inline src/client/components/Season.jsx:17]: Set these as constants in some config' file somewhere?

File: src/client/components/Season.jsx [hunk 7]
```diff
@@ -14,10 +14,10 @@ export const Season = () => {
...
+                <option className="dropdown-option" value="2">{`Coconut Mall`}</option>
```
Comment (glowkeeper) [inline src/client/components/Season.jsx:18]: If you had some constant object somewhere, you could use `value={competition.key}>{compeition.name}<` - that'd be _much_ more robust

**Review by glowkeeper — APPROVED** [review]
I've approved _with caveats_ - I reckon your codebase would benefit by defining some constants in some config file and by better use of objects ;)

File: src/client/components/Season.jsx (hunk 6 above)
Comment (bravint) [reply src/client/components/Season.jsx:17]: these are placeholders, they will be changed as we continue to merge frontend and backends together

**Pull request merged**